core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-system-service.yml

```diff
@@ -8,9 +8,9 @@
   vars:
     _packages:
       Debian:
-        - filebeat={{ filebeat_version }}
+        - filebeat={{ specification.filebeat_version }}
```

Comment:
Why it's taken from specification if we have that specified in the role's defaults?

Reply:
Yes, it looks like: https://github.com/epiphany-platform/epiphany/blob/v0.8/core/src/epicli/data/common/ansible/playbooks/roles/filebeat/defaults/main.yml